utils/doclint/cli.js

```diff
@@ -43,6 +43,7 @@ async function run() {
   documentation.filterForLanguage('js');
 
   // Patch README.md
+  /*
```

Comment:
Revert changes in this file.

Reply:
Ah my bad! Sorry that was accidental.. Thanks for observing that! Just pushed a new commit to revert these lines.

Reply:
Looks like this change is still present.